### PR TITLE
Moved Support to .org

### PR DESCRIPTION
**Pull request by MichaelRupertDev**
Merge after/at the same time as https://github.com/borgbackup/borgbackup.github.io/pull/20

### CODE REVIEW

File: docs/man_intro.rst [hunk 1]
```diff
@@ -65,4 +65,4 @@ SEE ALSO
 * Releases https://github.com/borgbackup/borg/releases
 * Changelog https://github.com/borgbackup/borg/blob/master/docs/changes.rst
 * GitHub https://github.com/borgbackup/borg
-* Security contact https://borgbackup.readthedocs.io/en/latest/support.html#security-contact
+* Security contact https://borgbackup.org/support/support.html#security-contact
```
Comment (ThomasWaldmann) [inline docs/man_intro.rst]: guess this points to nowhere, as you kept the security stuff in the docs?

**Comment by ThomasWaldmann** [comment]
btw, instead of fixing another docs issue, i guess it would be better to backport this to 1.1-maint branch after it is merged to master.

**Comment by MichaelRupertDev** [comment]
can do

**Comment by ThomasWaldmann** [comment]
thanks!

**Pull request merged**